packages/api/cms-api/src/public-upload/public-uploads.controller.ts

```diff
@@ -2,20 +2,20 @@ import { EntityManager } from "@mikro-orm/postgresql";
 import { Controller, Post, UploadedFile, UseInterceptors } from "@nestjs/common";
 import rimraf from "rimraf";
 
-import { PublicApi } from "../auth/decorators/public-api.decorator";
+import { DisableCometGuards } from "../auth/decorators/disable-comet-guards.decorator";
 import { PublicUploadFileUploadInterface } from "./dto/public-upload-file-upload.interface";
 import { PublicUpload } from "./entities/public-upload.entity";
 import { PublicUploadFileInterceptor } from "./public-upload-file.interceptor";
 import { PublicUploadsService } from "./public-uploads.service";
 
 @Controller("public-upload/files")
-@PublicApi()
+@DisableCometGuards()
```

Comment:
We should remove it here and as it's already set in the upload function. Also, setting the decorator on controllers/decorators could lead to unintentionally creating open routes